portfolio/src/main/webapp/index.html

```diff
@@ -80,10 +80,9 @@ <h2>Comments</h2>
       <form action="/data" method="post" target="_self">
         <textarea name="comment" placeholder="Comment..."></textarea>
         <br/><br/>
-        <input type="submit" value="Submit">
+        <input type="submit" value="Submit" >
       </form>
-      <p>Click below to see comments left by others!</p>
-      <button class="button" onclick="getCommentsUsingArrows()">Comments</button>
+        <script>getCommentsUsingArrows()</script>
```

Comment:
If you make the "IIFE" change I recommended above, I don't think you'll need this line.

If it does provide value, then nit: indentation should match the lines above and below.